src/schema.ts

```diff
@@ -601,6 +601,9 @@ export const typeDefs = gql`
     about: SubjectPageAbout
     metaDescription: String
     supportedLanguages: [String!]!
+    connectedTo: [String!]!
+    buildsOn: [String!]!
+    leadsTo: [String!]!
```

Comment:
Korleis skal du bruke disse i ndla-frontend? Bruke cacha subjects-liste for å rendre lenkene?
Et alternativ er å lage en ny type med path og navn og returnere liste med den. Då kan du rendre lenker uten å måtte gjøre jobb i ndla-frontend.